samtranslator/model/preferences/deployment_preference_collection.py

```diff
@@ -152,6 +152,67 @@ def deployment_group(self, function_logical_id):
 
         return deployment_group
 
+    def _process_alarms(self, preference, group):
```

Comment:
I see "group" is used only as an output parameter(lines 181,  & 184) I would prefer to remove it and make the call that way:
```
group.AlarmConfiguration = self._process_alarms(deployment_preference);
```

Reply:
Indeed and return None at line 172

Reply:
in this case, you might want to rename "process_alarms" to something more descriptive